File: bc/bea_cukai/report/laporan_pengeluaran_barang_perdokumen/laporan_pengeluaran_barang_perdokumen.js
// Copyright (c) 2016, PT. UNITEK MAS INDONESIA and contributors
// For license information, please see license.txt
/* eslint-disable */

frappe.query_reports["Laporan Pengeluaran Barang Perdokumen"] = {
	"filters": [
		{
			"fieldname":"jenis_dokumen_bc",
			"label":("Jenis Dokumen"),
			"fieldtype":"Select",
			"options":["BC.3.0","BC.2.5","BC.2.7","BC.2.6.1","BC.4.1","BA Pemusnahan"]
		},
				{
			"fieldname":"from_date",
			"label": __("Dari Tanggal"),
			"fieldtype": "Date",
			"default": frappe.datetime.add_months(frappe.datetime.get_today(), -1),
			"reqd": 1
		},
		{
			"fieldname":"to_date",
			"label": __("Sampai Tanggal"),
			"fieldtype": "Date",
			"default": frappe.datetime.get_today(),
			"reqd": 1
		}
		

			]
}

File: bc/bea_cukai/report/laporan_pengeluaran_barang_perdokumen/laporan_pengeluaran_barang_perdokumen.py
# Copyright (c) 2013, PT. UNITEK MAS INDONESIA and contributors
# For license information, please see license.txt

from __future__ import unicode_literals
import frappe

def execute(filters=None):
	columns = [{
    "fieldname": "naming_series",
    "label": _("Series"),
    "fieldtype": "select",
    "options": "SI-BC-OUT-.YYYY.-",
    "width": 300
},
]